core/kubernetes/main/k8s-aws.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+kustomizationFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+openhimConsoleVolumePath="${kustomizationFilePath}/openhim/volume/openhim-console/default.json"
+openhimApiUrl=''
+fhirServerUrl=''
+openhimConsoleUrl=''
+
+if [ "$1" == "up" ]; then
+    kubectl apply -k $kustomizationFilePath
+
+    openhimApiUrl=$(kubectl get service openhim-core-service -o=jsonpath={.status.loadBalancer.ingress[0].hostname})
+    fhirServerUrl=$(kubectl get service hapi-fhir-server-service -o=jsonpath={.status.loadBalancer.ingress[0].hostname})
```

Comment:
Surely similar techniques would work with minikube to? Could we not rather try update the original script to work with eks rather than having two very similar scripts. That will make it easier for us to maintain and also won't break the `./isntant.sh` script which will be the entry point for running all of this.